src/XrdHttp/XrdHttpProtocol.cc

```diff
@@ -1135,6 +1139,15 @@ int XrdHttpProtocol::Config(const char *ConfigFN, XrdOucEnv *myEnv) {
 //
    if (myEnv->Get("XrdCache")) hasCache = true;
 
+   if(strlen(xrdcorsLibPath) && LoadCorsHandler(&eDest, xrdcorsLibPath)) {
```

Comment:
Assuming you chose a pointe, then simply test for null, orif using  std::string, simply test for size() > 0.